packages/@okta/vuepress-site/docs/reference/api/brands/index.md

```diff
@@ -0,0 +1,1371 @@
+
+None.
+
+Passing an invalid `brandId` returns a `404 Not Found` status code with error code `E0000007`.
+
+Passing an invalid `themeId` returns a `404 Not Found` status code with error code `E0000007`.
+
+#### Use examples
+
+The following request removes the uploaded Theme logo.
+
+##### Request
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/brands/{brandId}/themes/{themeId}/logo"
+```
+
+##### Response
+
+```http
+HTTP/1.1 204 No Content
+```
+
+### Upload Theme Favicon
+
+<ApiOperation method="post" url="/api/v1/brands/{brandId}/themes/{themeId}/favicon" />
+
+Updates the favicon for your theme
+
+#### Request body
+
+| Property | Type | Description                                                                                                                                                                                               |
+|----------|------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `file`   | File | The file must be in PNG, or ICO format and must be in 1:1 ratio with maximum 512 x 512 dimensions |
+
+#### Response body
+
+Returns `201 Created` with [Favicon URL](#image-upload-response-object)
+
+Passing an invalid `brandId` returns a `404 Not Found` status code with error code `E0000007`.
+
+Passing an invalid `themeId` returns a `404 Not Found` status code with error code `E0000007`.
+
+Passing an invalid `file` returns a `400 Bad Request` status code with error code `E0000001`.
+
+#### Use examples
+
+The following request updates the Theme Favicon with the uploaded file.
+
+##### Request
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-F 'file=@/path/to/file' \
+"https://${yourOktaDomain}/api/v1/brands/{brandId}/themes/{themeId}/favicon"
+```
+
+#### Response
+
+```http
+HTTP/1.1 201 Content Created
+Content-Type: application/json
+```
+
+```json
+{
+  "url": "https://${yourOktaDomain}/bc/image/fileStoreRecord?id=fs09yfpj6PnWzgzIQ0g4"
+}
+```
+
+The following request shows invalid Favicon dimensions validations.
+
+##### Request
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-F 'file=@/path/to/invalid-file' \
+"https://${yourOktaDomain}/api/v1/brands/{brandId}/themes/{themeId}/favicon"
+```
+
+#### Response
+
+```http
+HTTP/1.1 400 Bad Request
+Content-Type: application/json
+```
+
+```json
+{
+  "errorCode": "E0000001",
+  "errorSummary": "Api validation failed: file",
+  "errorLink": "E0000001",
+  "errorId": "oae-71S2vP_TWWqdCVEndqHCw",
+  "errorCauses": [
+    {
+      "errorSummary": "Your selected image should be in 1:1 ratio for width and height. found 199 x 200, it should be 199 x 199 or 200 x 200"
```

Comment:
any chance we can fix this so that it's more readable? 

```suggestion
      "errorSummary": "Your selected image should be in a 1:1 ratio for width and height. Found 199 x 200. The image should be 199 x 199 or 200 x 200."
```

Reply:
updated the error message in monolith: https://github.com/okta/okta-core/pull/55310
@susanharper-okta @albertchen-okta